src/main/flight/mixer.c

```diff
@@ -523,7 +529,13 @@ static void applyMixerAdjustmentEzLand(float *motorMix, const float motorMixMin,
 
     // Upper throttle limit
     // range default 0.05 - 1.0 with ezLandingLimit = 5, no stick deflection -> 0.05
-    const float ezLandLimit = calcEzLandLimit(getMaxRcDeflectionAbs());
+#ifdef USE_GPS
+    const float speed = gpsSol.speed3d;
```

Comment:
Lot of users include GPS in their build so we need to check if available in another way - for example:

https://github.com/betaflight/betaflight/blob/47f3c31f54b5fe729db4a6c4cdc2030f2ffdf3d1/src/main/io/gps.c#L2521-L2522

Reply:
It's just a sanity check to make sure it compiles without USE_GPS, wasn't sure if gpsSol would be available then.
For now it appears like gpsSol.speed3d will stay at 0 if USE_GPS is defined but no GPS is connected, but that should not be taken for granted. That and a bunch of other GPS sanity checks are still TODO.

Reply:
Now it at least checks fora a GPS 3D fix before using the speed from GPS.